packages/core/admin/server/routes/authentication.js

```diff
@@ -5,19 +5,28 @@ module.exports = [
     method: 'POST',
     path: '/login',
     handler: 'authentication.login',
-    config: { auth: false },
+    config: {
+      auth: false,
+      middlewares: ['admin::rateLimit'],
+    },
   },
   {
     method: 'POST',
     path: '/renew-token',
     handler: 'authentication.renewToken',
-    config: { auth: false },
+    config: {
+      auth: false,
+      middlewares: ['admin::rateLimit'],
+    },
   },
   {
     method: 'POST',
     path: '/register-admin',
     handler: 'authentication.registerAdmin',
-    config: { auth: false },
+    config: {
+      auth: false,
+      middlewares: ['admin::rateLimit'],
+    },
```

Comment:
Is it relevant to add it here? I think it's the first registration, which can happen only once

Reply:
Also, we should add `registration-info` also as it takes the registration token as a param and can then allow to test if a registration token is valid and then use the register route